app/models/foreman_azure_rm/azure_rm.rb

```diff
@@ -183,14 +183,14 @@ def create_nics(args = {})
           pip = azure_network_service.public_ips.create(
               name:                        "#{args[:vm_name]}-pip#{nic}",
               resource_group:              args[:resource_group],
-              location:                    formatted_location,
+              region:                      formatted_region,
```

Comment:
This is fog's interface [property](https://github.com/fog/fog-azure-rm/blob/master/lib/fog/azurerm/models/network/network_interface.rb#L8), it should remain `location`